msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,618 @@
+---
+title: Add an Outlook Add-in to a Teams app
+description: Learn how to add an Outlook Add-in to a Teams app.
+ms.date: 01/11/2024
+ms.author: mosdevdocs
+author: rickki
+ms.topic: tutorial
+ms.localizationpriority: medium
+ms.subservice: m365apps
+---
+
+# Add an Outlook Add-in to a Teams app
+
+## Introduction
+
+Outlook Add-ins are web apps that extend the functionality of Outlook. Some of the things that can be done with an Outlook Add-in:
+
+- Read and write the content of email messages and meeting invitations, as well as responses, cancellations, and appointments.
+- Read properties of the user's mailbox.
+- Respond automatically to events, such as the sending of an email.
+- Integrate with external services including CRM and project management.
+- Add custom ribbon buttons or menu items to perform specific tasks.
+
+[Outlook Add-ins Overview](/office/dev/add-ins/outlook/outlook-add-ins-overview) which will give you information about what you can do with Outlook Add-ins, how they are structured, how they differ from older ways of extending Outlook, what platforms the add-in can run on (Windows, Mac, mobile, and the web), and how to get started creating one.
```

Comment:
Is this sentence needed? The link is covered in "See also" section, and you could also just inline it in the first instance of "Outlook Add-ins"in the article

Reply:
Looking again, I think this text works fine--it just needs a minor tweak so it reads as a proper sentence. How about:

**For more info,** [Outlook Add-ins Overview](/office/dev/add-ins/outlook/outlook-add-ins-overview) **provides** information about what you can do with Outlook Add-ins, how they are structured, how they differ from older ways of extending Outlook, what platforms the add-in can run on (Windows, Mac, mobile, and the web), and how to get started creating one.

Reply:
> Looking again, I think this text works fine--it just needs a minor tweak so it reads as a proper sentence. How about:
> 
> **For more info,** [Outlook Add-ins Overview](/office/dev/add-ins/outlook/outlook-add-ins-overview) **provides** information about what you can do with Outlook Add-ins, how they are structured, how they differ from older ways of extending Outlook, what platforms the add-in can run on (Windows, Mac, mobile, and the web), and how to get started creating one.

I added your version back in.